mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -585,6 +602,29 @@ object OptionGroup extends LazyLogging {
     }
   }
 
+  /** Options used to configure trace evaluation. */
+  case class Tracee(
+      trace: File,
+      expressions: List[String])
+      extends OptionGroup
+
+  object Tracee extends Configurable[Config.Tracee, Tracee] with LazyLogging {
+    def apply(tracee: Config.Tracee): Try[Tracee] = {
+
+      def validateExprs(lst: List[String]): Try[List[String]] =
+        if (lst.nonEmpty) Success(lst)
+        else Failure(new PassOptionException("Trace evaluation requires a nonempty list of expressions."))
+
+      for {
+        trace <- tracee.trace.toTry("trace.trace")
```

Comment:
Once this is a `SourceOption` you can validate at this point that you only hava JSON or ITF file.